design/07-ui-ux/09-errors.md

```
@@ -1,35 +1,90 @@
+空状態（プロファイル 0 件・要素 0 件）は [`04-dashboard.md`](04-dashboard.md) を参照。
+
+---
+
+## デバイス構成編集画面
 
-初回起動時（`recent.workspaces` が空の場合）、ダッシュボード表示前にワークスペース選択ダイアログを表示する。
+### バインディングが 0 件（新規作成直後）
+
+バインディングタブのリストが空の状態。案内テキストを表示する。
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)